docs/fundamentals/client/client.md

```diff
@@ -668,6 +663,20 @@ with Flow().add() as f, open('output.txt', 'w') as fp:
     )
 ```
 
+````{admonition} What errors can be handled by the callback
+:class: caution
+Callback can handle errors that are caused inside the Flow, it can be from an Executor raising an Exception or from the Gateway having connectivity errors with the Executors.
+It will not receive errors of connectivity between the Client and the Gateway.
+````
+
+### Transient fault handling with retries
+
+`client.post()` accepts `num_retries` and `retry_delay` parameters to control the capacity to retry requests when a transient connectivity error occurs.
+This can help to overcome transient network connectivity issues.
```

Comment:
it feels like we're hiding the retry mechanism on users. If it's meant to be the same for all protocols I'd decorate all `send_request` methods with the same decorator, otherwise say that in grpc it relies on grpc retry mechanism

Reply:
I will say